packages/webassemblyjs/test/index.js

```diff
@@ -15,7 +15,7 @@ function toArrayBuffer(buf) {
 describe("interpreter", () => {
   describe("wasm", () => {
     const testSuites = glob.sync(
-      "packages/WebAssembly/test/fixtures/**/module.wasm"
+      "packages/webassemblyjs/test/fixtures/**/module.wasm"
```

Comment:
@xtuc I was wondering why the total number of unit tests had dropped recently ;-)

Reply:
omg, that's a good catch!